internal/backends/winit/lib.rs

```diff
@@ -324,6 +330,10 @@ impl BackendBuilder {
                     return Err(PlatformError::NoPlatform);
                 }
             }
+            #[cfg(all(feature = "renderer-femtovg-wgpu", feature = "unstable-wgpu-24"))]
```

Comment:
nitpick:

```suggestion
            #[cfg(feature = "unstable-wgpu-24")]
```

unstable-wgpu-24 implies renderer-femtovg-wgpu